helm-chart/todoapp/values.yaml

```diff
@@ -0,0 +1,45 @@
+namespace: todoapp
+
+todoapp:
+  image:
+    repository: "ikulyk404/todoapp"
+    tag: "4.0.1"
+
+  secrets:
+    SECRET_KEY: "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
+    DB_NAME: "app_db"
+    DB_USER: "app_user"
+    DB_PASSWORD: "1234"
+    DB_HOST: "mysql-0.mysql-service.mysql.svc.cluster.local"
```

Comment:
Storing plaintext secrets in `values.yaml` is a security risk. Consider using a more secure method to manage sensitive data, such as using Kubernetes Secrets or a secret management tool to inject these values securely.